astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -634,10 +634,12 @@ def test_all(self):
         with pytest.raises(TypeError):
             np.all(self.q)
 
+    @pytest.mark.filterwarnings("ignore:`sometrue` is deprecated as of NumPy 1.25.0")
```

Comment:
I don't like removing `NUMPY_LT_1_25`. Makes it hard to grep and remove dead code when we eventually bump minversion to numpy 1.25. Can we at least leave `# NUMPY_LT_1_25` here for future grep?

Reply:
Unless numpy dev remove it first before that happens but I can't tell the future.

Reply:
👍 done